src/main/java/com/powsybl/openloadflow/util/ParameterConstants.java

```diff
@@ -21,7 +21,7 @@
     public static final SlackBusSelector SLACK_BUS_SELECTOR_DEFAULT_VALUE = new MostMeshedSlackBusSelector();
 
     public static final String THROWS_EXCEPTION_IN_CASE_OF_SLACK_DISTRIBUTION_FAILURE_PARAM_NAME = "throwsExceptionInCaseOfSlackDistributionFailure";
-    public static final boolean THROWS_EXCEPTION_IN_CASE_OF_SLACK_DISTRIBUTION_FAILURE_DEFAULT_VALUE = true;
+    public static final boolean THROWS_EXCEPTION_IN_CASE_OF_SLACK_DISTRIBUTION_FAILURE_DEFAULT_VALUE = false;
```

Comment:
I don't known if it should be the defautl behaviour. I think we did that because of network with multiple synchronous components. Very often there is no way to fully distribute slack on small synchronous components and we don't want the simulation to fail

Reply:
Yes but this modification, the default behavior is to log an error.

Reply:
Ok